### ISSUE TITLE
Large database payloads can fail

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/paper-land#250]
> Look for other places where this payload error can happen.

Inspecting the api, here is where this can happen again potentially:

- /api/spaces/:spaceName/programs (POST):  - When you create a new program from the camera page. Will happen if code is too large.
- /api/spaces/:spaceName/programs/add-premade-program (POST): Sends a single program to playground from Creator, part of the chunking from #188. If the program is too large, we will hit this.
- /api/creator/templates (PUT): Saves a new template for creator. Just like this issue, if the template is has ~200 components or more, we will hit this.
- /api/creator/templates/update/save (PUT): Again, if template is massive when you update it, this will happen.
- /api/creator/maintenance/updateSchema (POST): (Not really worried about this, its for development use)

Could also happen if the user tries to upload massive image/audio files.

The size limits are way bigger for receiving data from the server. Since our entire db is less than 1mb, we don't have to worry about this when loading data from the db.

In summary, one of these is from the original paper programs code and the rest are not likely enough to hit to prioritize right now (IMO). However this may be something @brettfiedler wants to track for project management. I think this is most likely to happen again for massive programs (where a single program has >200 components or tons of custom code).

_Originally posted by @jessegreenberg in https://github.com/phetsims/paper-land/issues/202#issuecomment-1814735146_

Noting that this can happen as described above. Will wait until we hit this scenario to act.
